src/index.jsx

```diff
@@ -109,7 +109,7 @@ function bootstrap() {
 function render(Component) {
   const root = document.getElementById('react');
   const { configuration } = store.getState();
-  const isGoogleAuthority = configuration.configObject.authorityEndpoint.startsWith(
+  const isGoogleAuthority = configuration.configObject.authorityEndpoint?.startsWith(
```

Comment:
Without this change, the main page would fail to render if the server returned an error state. The field `authorityEndpoint` is optional.